### DISCUSSION TITLE
Open a ticket in Gitlab

### DISCUSSION THREAD

**Discussion by gohabsgo2142** [colinlienard/gitlight#282] [Ideas]
My gitlab is a private URL. When I select the ticket to open the version, it direct me to gitlab normal url but I do use a specific URL. Is it possible for when I click on my notification, it open my specific URL ? 

Thanks !

**Comment by colinlienard** [comment]
What do you mean exactly by selecting the ticket to open the version? Also I'm not sure what "specific url" means here, do you mean GitLab self-hosted?
